docs/notebooks/openai_gym_lunar_lander.pct.py

```diff
@@ -0,0 +1,253 @@
+# ---
+# jupyter:
+#   jupytext:
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.13.4
+#   kernelspec:
+#     display_name: Python 3 (ipykernel)
+#     language: python
+#     name: python3
+# ---
+
+# %% [markdown]
+# # Trieste meets OpenAI Gym
+#
+# This notebook demonstrates how to use Trieste to apply Bayesian optimization to a problem that is slightly more practical than classical optimization benchmarks shown used in other tutorials. We will use OpenAI Gym, which is a popular toolkit for reinforcement learning algorithms.
+#
+# Concretely, we are going to take the [Lunar Lander](https://gym.openai.com/envs/LunarLander-v2/) environment, define a search space and describe it as an optimization problem, and use Trieste to find an optimal solution for the problem. And hopefully avoid too many landers crashing on the Moon surface along the way.
+
+# %%
+import tensorflow as tf
+import numpy as np
+import trieste
+import gpflow
+
+
+import gym
+ENV_NAME = "LunarLander-v2"
```

Comment:
in other notebooks we haven't been using upper case for constants, so lets be consistent :)

Reply:
Converted to lower case